spec/controllers/idv/in_person/public/address_search_controller_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Idv::InPerson::Public::AddressSearchController do
+  include Rails.application.routes.url_helpers
+
+  describe '#index' do
+    subject(:action) do
+      post :index,
+           params:
+            { address: '100 main' }
+    end
+    context 'with feature flag off' do
+      before do
+        allow(IdentityConfig.store).to receive(:in_person_public_address_search_enabled).
+          and_return(false)
```

Comment:
I think you need to call `Rails.application.reload_routes!` before/after to reload the specs. We've had lots of issues with test pollution with this in the past, so I'd be careful

Reply:
Yeah, I think the preference is to use `check_or_render_not_found` to avoid that, ex: https://github.com/18F/identity-idp/blob/9a6deb89e2b7a09d55145f916f88f85f9ef0438d/app/controllers/idv/in_person/ready_to_verify_controller.rb#L8